public/less/_repository.less

```diff
@@ -1489,3 +1489,21 @@
 .repository .header-wrapper .ui.tabular .fa {
 	margin-right: 5px;
 }
+
+.hashtag.page {
```

Comment:
Would it be worth putting our changes to `.less` files in different files, to avoid merge conflicts with upstream?